ide-common/src/main/java/org/digma/intellij/plugin/document/DocumentInfoContainer.java

```diff
@@ -99,7 +110,7 @@ private List<String> getObjectIds() {
         return this.documentInfo.getMethods().values().stream().flatMap((Function<MethodInfo, Stream<String>>) methodInfo -> {
             var ids = new ArrayList<String>();
             ids.add(methodInfo.idWithType());
```

Comment:
I think here you need only ids not ids with type but i'm not sure

Reply:
`idWithType` is needed here as it requires for last section of insights on the screen with specific type like "span:some-insight..":
<img width="394" alt="image" src="https://user-images.githubusercontent.com/117861250/209567368-4f6fbb60-fb52-41c4-8e23-1be2515a50c4.png">